_studio/mfx_lib/shared/src/mfx_common_int.cpp

```diff
@@ -550,7 +550,8 @@ mfxStatus CheckDecodersExtendedBuffers(mfxVideoParam const* par)
                                                                MFX_EXTBUFF_FEI_PARAM};
 
     static const mfxU32 g_decoderSupportedExtBuffersHEVC[]  = {
-                                                               MFX_EXTBUFF_HEVC_PARAM
+                                                               MFX_EXTBUFF_HEVC_PARAM,
+															   MFX_EXTBUFF_DEC_VIDEO_PROCESSING
```

Comment:
Please replace tabs with spaces.

Reply:
Done, Thanks!

Reply:
> Done, Thanks!
I removed the line later, because in umc_h264_va_packer.cpp, no such line. I am not sure about it.